### DISCUSSION TITLE
change mode when connecting/disconnecting from Wi-Fi network

### DISCUSSION THREAD

**Discussion by Kaynberg** [owntracks/android#1846] [Q&A]
Hello, sorry for my English.
Is it possible to implement the function of changing the tracking mode when connecting to a Wi-Fi network?
For example, when leaving home, the smartphone disconnects from the home network and the owntracks changes the mode to "Move mode: Frequent location ||>"
Thanks.

**Comment by jpmens** [comment]
This is possible with other tools. I think people use _tasker_ for accomplishing this.
